tests/robot-tests/tests/admin/bau/create-amendment-check-permalink.robot

```diff
@@ -0,0 +1,198 @@
+*** Settings ***
+Resource  ../../libs/admin-common.robot
+Resource  ../../libs/common.robot
+Library  ../../libs/admin_api.py
+Library  Collections
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       Run Keywords  user signs in as bau1
+Suite Teardown    Run Keywords  user closes the browser
+
+*** Variables ***
+${DETAILS_HEADING}  Academic Year 2020/21 (not Live)
+${AMEND_RELEASE_HEADING}  Academic Year 2020/21 (Live - Latest release)
+${PUBLICATION_NAME}  Permalink Test Publication %{RUN_IDENTIFIER}
+${TOPIC_NAME}  %{TEST_TOPIC_NAME}
+${SUBJECT_NAME}   Dates test subject
+
+*** Test Cases ***
+create publication & releae
+  ${PUBLICATION_ID}=  user creates test publication via api  ${PUBLICATION_NAME}
```

Comment:
We use 4 spaces for indentation. This needs fixing elsewhere in the test suite too.